hardly/handlers/distgit.py

```diff
@@ -75,11 +75,14 @@ def run(self) -> TaskResults:
             package_config=self.package_config,
             upstream_local_project=self.local_project,
         )
-
+        dg_mr_info = f"""Info for package maintainer:
+This MR has been automatically created from source-git MR {self.mr_url}
+You are expected to run the CI Pipeline manually
+via 'Pipelines -> Run pipeline' after reviewing the change."""
```

Comment:
What about this?

```suggestion
        dg_mr_info = f"""---\n###### Info for package maintainer

This MR has been automatically created from [this source-git MR]({self.mr_url}).
You are expected to run the CI Pipeline manually
via `Pipelines → Run pipeline` **after** reviewing the change."""
```

Looks like:
![](https://i.imgur.com/4Ujy5JT.png)

Reply:
+1 formatting